components/list/list-controls.js

```diff
@@ -19,11 +19,13 @@ export class ListControls extends SelectionControls {
 			:host {
 				--d2l-selection-controls-background-color: var(--d2l-list-controls-background-color);
 				--d2l-selection-controls-padding: var(--d2l-list-controls-padding, 18px);
-				z-index: 6; /* must be greater than d2l-list-item-active-border */
 			}
 			:host([no-sticky]) {
 				z-index: auto;
 			}
+			:host([_scrolled]) {
+				z-index: 6;  /* must be greater than d2l-list-item-active-border and d2l-button-add */
+			}
```

Comment:
Otherwise when hover/focused the add-button gets cut off by the controls. I think the scrolled case is the biggest issue here but let me know if there are other cases.